omero/sysadmins/limitations.rst

```diff
@@ -79,6 +79,23 @@ This primarily affects the following file formats:
 This issue can be avoided by pre-generating pyramidal OME-TIFF images as
 described above.
 
+.. _ngff_limitations:
```

Comment:
I think not. The OME-NGFF is a format, and imho should live under ``File format support``. Also, the depth paragraph is formulated intentionally to be not OME-NGFF only specific.

Reply:
> Since we are creating a link to a set of limitations,

created the link to be able to link to this section directly from e.g. release announcement, maybe this was not the right idea ?

Reply:
Definitely a good idea to create a reference URL